ood-portal-generator/spec/ood_portal_generator_spec.rb

```diff
@@ -35,11 +35,11 @@
       expect(described_class.debian?).to eq(false)
     end
 
-    it 'returns true for Ubuntu 20.04' do
+    it 'returns true for Ubuntu 24.04' do
       os_release = <<~EOS
         ID=ubuntu
         ID_LIKE=debian
-        VERSION_ID="20.04"
+        VERSION_ID="24.04"
```

Comment:
While this is fine - I'm happy to merge it here, the version doesn't seem to even be under test. In any case - it's fine, just noting that we likely shouldn't update this test given the id will always increment eventually in the real world, but doesn't need to update here as time goes on.